### ISSUE TITLE
murmurhash2_x86_32 return 0 for strings length multiply of 4

### ISSUE THREAD

**Issue by Aung-Myint-Thein** [MartinHelmut/number-generator#78]
```
const murmurhash2_x86_32 = require('number-generator/lib/murmurhash2_x86_32');
murmurhash2_x86_32("A"); // returns 636687721
murmurhash2_x86_32("AA"); // returns 3621601297
murmurhash2_x86_32("AA"); // returns 3621601297
murmurhash2_x86_32("AAA"); // returns 2474210034
murmurhash2_x86_32("AAAA"); // returns 0
murmurhash2_x86_32("AAAAA"); // returns 2462844500
murmurhash2_x86_32("AAAAAA"); // returns 263112632
murmurhash2_x86_32("AAAAAAA"); // returns 433066591
murmurhash2_x86_32("AAAAAAAA"); // return 0
```

Maybe because of 

https://github.com/MartinHelmut/number-generator/blob/f112dec1131769478825fedfa8b737d3a703fd8c/src/murmurhash2_x86_32.js#L76

**Comment by MartinHelmut** [comment]
@Aung-Myint-Thein Thank you for reporting this issue! 🎉 I will add new test cases for that and fix that bug immediately.

**Comment by MartinHelmut** [comment]
Fixed with version 2.1.5 📣
Other numbers did not change, so this is non breaking. Thanks again for reporting @Aung-Myint-Thein  !
